components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -1554,8 +1560,29 @@ - (void)setTrackingScrollView:(UIScrollView *)trackingScrollView {
     }
   }
 
+  CFTimeInterval duration = kTrackingScrollViewDidChangeAnimationDuration;
+  CAMediaTimingFunction *timingFunction =
+      [CAMediaTimingFunction functionWithName:kCAMediaTimingFunctionEaseInEaseOut];
+
   void (^animate)(void) = ^{
+    self->_isAnimatingTrackingScrollViewChange = YES;
+
+    [CATransaction begin];
+#if TARGET_IPHONE_SIMULATOR
+    [CATransaction setAnimationDuration:duration * [self fhv_dragCoefficient]];
```

Comment:
what's the reason for the simulator to have a different animation duration?

Reply:
This is so that the "Slowed animations" Simulator debug flag will impact these animation durations as well.